Assembus/Assets/Scripts/Shared/DataExport.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System.IO;
+using System.Xml.Linq;
+using Models.Project;
+using Services.Serialization;
+using UnityEngine;
+
+namespace Shared
+{
+    public static class DataExport
+    {
+        private static readonly ProjectManager ProjectManager = ProjectManager.Instance;
+
+        /// <summary>
+        ///     Export assembly data to a given file
+        /// </summary>
+        /// <param name="exportPath">Path to save the xml file to</param>
+        /// <returns>A boolean indicating the success and an error message in case of failure</returns>
+        public static (bool, string) ExportData(string exportPath)
+        {
+            // Save chosen file name
+            ProjectManager.CurrentProject.ExportFileName = Path.GetFileName(exportPath);
```

Comment:
New export name doesn't get saved automatically to ProjectConfig!?

Reply:
@luismoser99 I'm sorry but I don't understand what you mean exactly.

Reply:
I am required to additionally save the actual project after exporting the data in order to save my previous export-filename to the projectConfig file. Is this the intended behaviour?